src/ui/utils/prefs.js

```diff
@@ -43,6 +43,7 @@ pref("devtools.features.transcriptHover", false);
 pref("devtools.features.widgetHover", false);
 pref("devtools.features.reactDevtools", false);
 pref("devtools.features.smoothPlayback", true);
+pref("devtools.features.videoPlayback", true);
```

Comment:
I don't think we want to enable this by default (yet)

Reply:
Works for me!